README.rst

```diff
@@ -173,6 +169,4 @@ the `Tsunami programme`_ created by `sociomantic labs GmbH`_.
 .. _All: https://github.com/sociomantic-tsunami/libdrizzle-redux/releases/
 .. _here: https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-user.rst
 .. _compiling.rst: https://github.com/andreas-bok-sociomantic/libdrizzle-redux/blob/v5.4.x/docs/compiling.rst
-.. _Tsunami programme: https://github.com/sociomantic-tsunami
-.. _sociomantic labs GmbH: https://www.sociomantic.com
 .. _Neptune: https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-user.rst#contributing-to-a-neptune-versioned-library
```

Comment:
Not strictly related to this PR but the neptune links are broken. Should be:
`https://github.com/sociomantic-tsunami/neptune/blob/v0.x.x/doc/library-user.rst`
and
`https://github.com/sociomantic-tsunami/neptune/blob/v0.x.x/doc/library-user.rst#contributing-to-a-neptune-versioned-library`

Reply:
:scream: I will make a separate PR to fix them